wiki/osu!wiki_Contribution_Guide/Common_Mistakes/en.md

```diff
@@ -0,0 +1,30 @@
+# Common Mistakes
+
+*Main page: [osu!wiki Contribution Guide](/wiki/owcg).*
+
+## All of my commits say "Updated en.md"
+
+Or with other locale files followed by `.md`. This can also be phrased as, "Someone told me to use better commit messages."
+
+This means you didn't use good commit names when you were able/suppose to or you had attempted to move/edit multiple files using the GitHub web interface. Remember, commit names are supposed to be meaningful so anyone can understand what you changed without having to look at the GitHub diff themselves but be short enough to fit the character limit of 72 characters.
+
+To resolve this (for both online and local editors):
+
+1. Create a new branch. Name it the same thing as your problematic branch and append a number, or whatever you want to name it as.
+2. Create a pull request.
+3. Click the `compare across forks` link, if they the `base fork` and `head fork` dropdown menus aren't present.
+4. Change `head fork` to your fork.
+5. Change `compare` to your problematic branch.
+6. Change `base fork` to your fork.
+7. Change `base` to your new branch.
```

Comment:
the order of 4, 5, 6, 7 can be changed. ¯\\_(ツ)_/¯

Reply:
the order shouldn't be changed, otherwise GitHub will try to be smart may try to do something weird... or at least that was what happened to me when I wrote that 16 step guide for Mock